CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v18.20.0
+
+Kubernetes API Version: 1.18.20
+
```

Comment:
Thanks for sending the PR! I noticed that the changes from the python-base submodule and the master branch weren't included. Could you first send a PR to update python-base submodule in the master branch; and then in this PR, [pull the updated master branch](https://github.com/kubernetes-client/python/blob/master/devel/release.md#create-or-update-release-branch) before running the rest steps?